Library/Homebrew/test/cask/audit_spec.rb

```diff
@@ -542,7 +556,7 @@ def include_msg?(messages, msg)
       let(:cask) { instance_double(Cask::Cask) }
 
       it "does not fail" do
-        expect(cask).to receive(:version).and_raise(StandardError.new)
```

Comment:
Once [`check_blacklist` was moved to the top of the checks](https://github.com/Homebrew/brew/pull/6435/files#diff-de813670c0d585d8a926b428a83c05b9R34), this failed as the cask received `:token` before version. I believe this still test the same behaviour by switching the expectation to `:token`.

Reply:
The messages seem wrong on all of these tests. `"does not fail"`, but we actually assert that it should fail.

Reply:
Since it didn't add this test, do you want me to fix them here, or follow up with another PR?

Reply:
A follow-up PR will be quicker to review.